code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -488,3 +489,32 @@
 		"/obj/item/weapon/mop",
 		"/obj/item/weapon/storage/bag/trash")
 
+/obj/item/weapon/storage/belt/leather
+	name = "leather belt"
+	desc = "Binds when equipped<br>Waist - Leather<br><span class='good'>+4 Strength</span><br><span class='good'>+4 Stamina</span><br>Requires Level 18"
+	icon_state = "leather"
+	item_state = "leather"
+	storage_slots = 2 //no pouches, no straps, just a waistband
+	canremove = 0 //crafted BoE gear
+
+/obj/item/weapon/storage/belt/leather/equipped(mob/living/M, slot)
+	..()
+	if(!istype(M))
+		return
+	if(slot == slot_belt)
+		desc = replacetext(desc, "Binds when equipped", "Soulbound")
+		M.maxHealth += 4
```

Comment:
this too, bizarre, nothing else in the game increases health and giving punch damage for wearing a belt is just not immersions friendly